app/controllers/groups_controller.rb

```diff
@@ -9,12 +9,15 @@ def index
   def show
     @post = Post.new
     @group = Group.friendly.find(params[:id])
+    @users = @group.users
```

Comment:
@nikitr I don't think you're using `@user` variable. And, you don't need to set it here. `@group.users` will do in the html!